code/datums/diseases/advance/symptoms/stinky.dm

```diff
@@ -0,0 +1,42 @@
+/*
+//////////////////////////////////////
+Stinky Syndicate
+	Somewhat Noticable.
+	Increases resistance.
+	Increases stage speed.
+	Very transmittible.
+	High Level.
+Bonus
+	Forces a spread type of AIRBORNE
+	with extra range!
+//////////////////////////////////////
+*/
+
+/datum/symptom/stinky
+	name = "Stinky Syndicate"
+	desc = "A highly resistant virus that causes rapid secretion of body odors that spread to nearby hosts."
+	stealth = 2
+	resistance = 10
+	stage_speed = 4
+	transmittable = 4
+	level = 666
```

Comment:
Intended?

Reply:
I assume I need a level. If I nerd a level I need one that doesn't exist so it can't be created.

The stats will definitely be rebalanced based on how well production testing goes.